references/segmentation/train.py

```diff
@@ -36,11 +39,12 @@ def get_transform(train, args):
         weights = torchvision.models.get_weight(args.weights)
         trans = weights.transforms()
 
-        def preprocessing(img, target):
+        def preprocessing(sample):
+            img, target = sample
             img = trans(img)
-            size = F.get_dimensions(img)[1:]
+            size = F.get_image_size(img)
```

Comment:
Reminder to not use the `get_image_size()` transform. It returns (w, h) instead (h, w). See #6581
```suggestion
            size = F.get_dimensions(img)[1:]
```